server/src/main/java/org/eclipse/openvsx/UserAPI.java

```diff
@@ -209,7 +209,7 @@ public CsrfTokenJson getCsrfToken(HttpServletRequest request) {
 
         var namespace = repositories.findNamespace(name);
         var userMembership = repositories.findMembership(user, namespace);
-        if (userMembership != null && userMembership.getRole().equals(NamespaceMembership.ROLE_OWNER)) {
+        if ((user.getRole() != null && user.getRole().equals("admin")) || (userMembership != null && userMembership.getRole().equals(NamespaceMembership.ROLE_OWNER))) {
```

Comment:
`UserData.ROLE_ADMIN.equals(user.getRole())` (this returns false if `user.getRole()` is null)